.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,141 @@
+  - [Issue](#issue)
+  - [Self-review](#self-review)
+  - [Pull-request](#pull-request)
+  - [Test](#test)
+- [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
+  - [Submit an inquiry request on github](#submit-an-inquiry-request-on-github)
+  - [Ask in the kirby-guild slack channel](#ask-in-kirby-guild)
+
+
+## The contribution model
+<!-- This section has the purpose of explaining why we do it like we do, in order to have people 
+symphatize with the process -->
+- What are we trying to achieve with this contribution model
+- General description of the contribution model
+
+<!-- The following "I have/would" sections has the role of easily guiding the reader 
+on how to do their specific task, and explain them what the process is for that task -->
+### I have a feature request 
+- Submit issue using the feature request template
+  - please read "[the good: issue](#issue)"
+- Contact kirby-guild to get it tech refined 
+- If you would like to implement it:
+  - please read the following before continuing: 
+    - "[Setup environment and tooling for contribution](#setup-environment-and-tooling-for-contribution)" to get properly setup 
+    - "[Creating pull-requests](#creating-pull-requests)" to get informed on how to most easily get your changes merged
+  - assign yourself to the issue to signal to others you are working on it
+  - if you at any point have questions while solving the issue please see: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
+
+### I have an enhancement 
+- Submit issue using the enhancement request template
+  - please read "[the good: issue](#issue)"
+- Contact kirby-guild to get it tech refined 
+- If you would like to implement issue: 
+  -please read: 
+    - "[Setup environment and tooling for contribution](#setup-environment-and-tooling-for-contribution)" to get properly setup 
+    - "[Creating pull-requests](#creating-pull-requests)" to get informed on how to most easily get your changes merged
+  - assign yourself to the issue to signal to others you are working on it
+  - if you at any point have questions while solving the issue please see: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
+
+### I would like to report a bug 
+- Submit issue using the bug template  
+  - please read "[the good: issue](#issue)"
+- Contact kirby-guild to get it verified 
+  - we verify because it might be a feature not a bug 🐛
+- If you would like to implement issue: 
+  -please read: 
+    - "[Setup environment and tooling for contribution](#setup-environment-and-tooling-for-contribution)" to get properly setup 
+    - "[Creating pull-requests](#creating-pull-requests)" to get informed on how to most easily get your changes merged
+  - assign yourself to the issue to signal to others you are working on it
+  - if you at any point have questions while solving the issue please see: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
```

Comment:
On one hand it's nice to have everything contained within each type (comp/enhancement/bug) on the other hand it's pretty WET - could we merge and move `If you would like to implement...` below 3 types? 🤔

Reply:
Yeah tought that myself. I have tried to write out the feature request part (which I've now renamed to component request), and will try to write out both the bug and enhancement part - while trying to format it to be more dry. 
But it's a balance between "here is a nice ordered checklist you should just follow"  and what I guess we can call "section-hopping" :)